platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/GossipEvent.java

```diff
@@ -261,7 +257,34 @@ public int getMinimumSupportedVersion() {
      */
     @Override
     public String toString() {
-        return EventStrings.toMediumString(this);
+        final StringBuilder stringBuilder = new StringBuilder();
```

Comment:
is there a test case that asserts the String is equivalent to what the old `EventStrings.toMediumString(this)` was generating, or at the very least as expected?

Reply:
These strings are used for debugging only. It might be worthwhile to compare the old result to the new manually as a sanity check, but adding a full unit test is probably not necessary. Debug strings like this tend to change over time as people modify and tweak them for solving new bugs.

Reply:
The new string produced is intentionally modified